docs/_guide/wallets/encrypting-and-storing.md

```diff
@@ -0,0 +1,17 @@
+# Encrypting and storing wallets
+
+## Creating a wallet and storing an encrypted JSON wallet on disk
+
+You can also manage a wallet using [JSON wallets](https://cryptobook.nakov.com/symmetric-key-ciphers/ethereum-wallet-encryption) that are securely encrypted and stored on the disk. This makes it easier to manage multiple wallets, especially for testing purposes.
+
+You can create a random wallet and, at the same time, encrypt and store it. Then, later, you can recover the wallet if you know the master password:
+
+[@code:typescript](./packages/wallet-manager/src/wallet-manager.test.ts#typedoc:wallet-manager-create)
```

Comment:
Is this related to storing/retrieving wallets to/from disk? I'm not sure how the support for these disk-stored JSON-encrypted wallets is at the moment in the TS SDK, and this example seems not to be doing anything related to it 🤔